### PR TITLE
[update] added partials for installing psql

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Added installing psql to install sections so that the users have all the information to install, connect, and set up a DB on one page.

PS: I was not able to build locally to check the changes because i kept getting:
<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 0x1013221f5 node::Abort() (.cold.1) [/Users/rajakavithakodhandapani/.nvm/versions/node/v14.19.1/bin/node]
 2: 0x1000b6399 node::Abort() [/Users/rajakavithakodhandapani/.nvm/versions/node# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: install/installation-macos.md [hunk 1]
```diff
@@ -112,12 +117,12 @@ section.
 
 </procedure>
 
+<MacPorts />
```
Comment (charislam) [inline install/installation-macos.md:120]: See comment for Homebrew.

**Review by charislam — CHANGES_REQUESTED** [review]
To make sure we are reusing content and not creating partials that are only used once, we might want to edit the [psql install page](https://docs.timescale.com/timescaledb/latest/how-to-guides/connecting/psql/) to make use of the new partials.

I built this locally and didn't see any errors, so I haven't been able to replicate the build issues yet. Let me know if they persist and I can do some more debugging!

**Comment by Loquacity** [comment]
Seems to be missing from the RHEL and Arch instructions, too.

**Comment by Rajakavitha1** [comment]
> Seems to be missing from the RHEL and Arch instructions, too.

Yes!!! that is right @Loquacity . The original install psql content does not have instructions for those.  I am going to spin up a RHEL and Arch instance install psql and then add the partials.

File: install/installation-macos.md [hunk 2]
```diff
@@ -80,6 +83,8 @@ You can use Homebrew to install TimescaleDB on macOS-based systems.
 
 </procedure>
 
+<Homebrew />
```
[the remark this answers is not in the document]
Comment (Rajakavitha1) [reply install/installation-macos.md:89]: added the intro sentence for psql

File: install/installation-macos.md (hunk 1 above)
Comment (Rajakavitha1) [reply install/installation-macos.md:126]: updated

File: install/installation-macos.md [hunk 3]
```diff
@@ -117,12 +120,14 @@ section.
...
+When you have PostgreSQL and TimescaleDB installed, you can connect to it from
+your local system using the `psql` command-line utility.
+
 <MacPorts />
 
 ## Set up the TimescaleDB extension
-
-When you have PostgreSQL and TimescaleDB installed, you can connect to it from
-your local system using the `psql` command-line utility.
+Connect to it PostgreSQL from your local system using the `psql` command-line
```
Comment (Loquacity) [inline install/installation-macos.md:129]:
```suggestion
Connect to PostgreSQL from your local system using the `psql` command-line
```

**Review by Loquacity — APPROVED** [review]

**Comment by Rajakavitha1** [comment]
> To make sure we are reusing content and not creating partials that are only used once, we might want to edit the [psql install page](https://docs.timescale.com/timescaledb/latest/how-to-guides/connecting/psql/) to make use of the new partials.
> 
> I built this locally and didn't see any errors, so I haven't been able to replicate the build issues yet. Let me know if they persist and I can do some more debugging!

Yes!!! I agree. I will work on the psql client install instructions for arch Linux and RHEL and then use the partials in the install psql page. Is that okay?

**Review by charislam — APPROVED** [review]

**Pull request merged**